float64 error
float64  velocity
---
string errorFeed